src/backend/glutin/mod.rs

```diff
@@ -287,9 +281,50 @@ unsafe impl Backend for GlutinBackend {
 
     #[inline]
     unsafe fn make_current(&self) {
-        let mut gl_window_takeable = self.borrow_mut();
-        let gl_window = Takeable::take(&mut gl_window_takeable);
-        let new_gl_window = gl_window.make_current().unwrap();
-        Takeable::insert(&mut gl_window_takeable, new_gl_window);
+        let pair = self.borrow();
+        pair.context.make_current(&pair.surface).unwrap();
     }
 }
+
+#[cfg(feature = "simple_winit_window")]
+/// Create a new winit Window and create a glium Display for it.
+///
+/// This function is meant as an easy way to get started> If you require more
+/// flexibility, you need to initialize the window and context yourself.
+/// Have a look at the source of this function as an example on how to do that.
+pub fn simple_winit_window(event_loop: &winit::event_loop::EventLoop<()>, window_title: &str) -> (winit::window::Window, Display<glutin::surface::WindowSurface>) {
```

Comment:
I think this function is great, but ideally there would additionally be a way to adjust the window size. E.g. by letting people pass in the window builder. I think it would be best to have that either in a separate function, or via the builder pattern, i.e. `SimpleWindowBuilder::new()` and `SimpleWindowBuilder::window_builder(...)`, and `SimpleWindowBuilder::build()` etc. This can be something for the future though, after this PR is merged.

Reply:
I went ahead and added a `SimpleWindowBuilder` with 2 methods for specifying title and size, as well as one for replacing the winit `WindowBuilder`. Because of this `simple_winit_window` isn't really necessary anymore and I've removed it and changed the tutorials to reflect that. Thought it might be good to have a builder from the start so that we can extend it without breaking existing code.